Resources/Prototypes/_RMC14/Entities/Clothing/OuterClothing/marine_armor.yml

```
@@ -347,6 +347,31 @@
       Classic: _RMC14/Objects/Clothing/OuterClothing/Armor/m3/co/classic.rsi
       Urban: _RMC14/Objects/Clothing/OuterClothing/Armor/m3/co/urban.rsi
 
+- type: entity
+  parent: CMArmorM2MP
+  id: RMCArmorM3LCO
+  name: M3-L pattern commanding officer armor
+  description: A lightweight variant of M3 Pattern Commanding Officer Armor. A Robust, well-polished suit of armor for the Commanding Officer. Custom-made to fit its owner with special straps to operate a smartgun. Show those Marines who's really in charge.
+  components:
+  - type: Sprite
+    sprite: _RMC14/Objects/Clothing/OuterClothing/Armor/m3/co-light/jungle.rsi
+  - type: CMArmor
+    armor: 30
+    bio: 20
+    explosionArmor: 15
+  - type: SmartGunArmor
+  - type: Storage
+    maxItemSize: Small
+    grid:
+    - 0,0,5,1 # 3 slots
```

You can drop a ton of lines by inheriting from the CO armor and only modify the sprites
```suggestion
  parent: RMCArmorM3CO
  id: RMCArmorM3LCO
  name: M3-L pattern commanding officer armor
  description: A variant of M3 Pattern Commanding Officer Armor. A Robust, well-polished suit of armor for the Commanding Officer. Custom-made to fit its owner with special straps to operate a smartgun. Show those Marines who's really in charge.
  components:
  - type: Sprite
    sprite: _RMC14/Objects/Clothing/OuterClothing/Armor/m3/co-light/jungle.rsi
```